### PR TITLE
implement auto-update

### PR DESCRIPTION
**Pull request by solomoncyj**

### CODE REVIEW

**Comment by solomoncyj** [comment]
any recomedatons?

File: Microsoft.PowerShell_profile.ps1 [hunk 1]
```diff
@@ -13,6 +13,24 @@
...
 ### more information about execution policies, run Get-Help about_Execution_Policies.
 
+#check for updates
+try{
+    $url = "https://raw.githubusercontent.com/ChrisTitusTech/powershell-profile/main/Microsoft.PowerShell_profile.ps1"
+    $oldhash = Get-FileHash $PROFILE
+    Invoke-RestMethod $url -OutFile "$env:temp/Microsoft.PowerShell_profile.ps1"
+    $newhash = Get-FileHash "$env:temp/Microsoft.PowerShell_profile.ps1"
+    if ($newhash -ne $oldhash) {
+        Get-Content "$env:temp/Microsoft.PowerShell_profile.ps1" | Set-Content $Profile
```
Comment (pixincreate) [inline Microsoft.PowerShell_profile.ps1:23]:
```suggestion
        Get-Content "$env:temp/Microsoft.PowerShell_profile.ps1" | Set-Content $PROFILE
```

**Review by pixincreate — APPROVED** [review]
Other than that looks good to me!

**Comment by solomoncyj** [comment]
@ChrisTitusTech ready to merge

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**